toucan_connectors/toucan_connector.py

```diff
@@ -43,6 +43,11 @@ def is_oauth2_connector(cls):
     return hasattr(cls, '_auth_flow') and getattr(cls, '_auth_flow') == 'oauth2'
 
 
+def get_form(cls):
```

Comment:
Maybe it's confusing to have the same name for the _connector_ config form (oauth2), and the _connection_ config form.
It can wait for the coming refacto I think

Reply:
I changed the name to get_connector_config_form. maybe get_connector_admin_form maybe ?